File: Lab3/pzpi-22-6-holovashenko-illia-lab3/src/app/core/models/sensor.dto.ts
export interface SensorDto {
  sensorId: number;
  sensorName: string;
  sensorType: string;
  sensorStatus: 'enabled' | 'disabled';
  buildingId: number;
  dateAdded: string;         // ISO формат, наприклад: '2025-06-04T12:00:00.000Z'
  lastDataReceived: string;  // ISO формат
}

File: Lab4/pzpi-22-6-holovashenko-illia-lab4/app/src/main/java/ua/nure/holovashenko/flameguard_mobile/data/dto/SensorDto.kt
package ua.nure.holovashenko.flameguard_mobile.data.dto

data class SensorDto(
    val sensorId: Int? = null,
    val sensorName: String,
    val sensorType: String,
    val sensorStatus: String,
    val lastDataReceived: String?,
    val dateAdded: String?,
    val buildingId: Int
)
